docs/reference/supported-models.md

```
@@ -123,14 +123,22 @@ Used by `totem sync` for indexing chunks into LanceDB.
 
 ## Totem Defaults
 
-Current defaults configured in `totem.config.ts` and `packages/cli/src/commands/init.ts`:
+Current defaults configured in `totem.config.ts` (this repo) and stamped into
+generated consumer configs by `totem init`
+(`packages/cli/src/commands/init-detect.ts`):
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)